messages/dev.generate.flag.md

```diff
@@ -158,60 +158,60 @@ Do you want to use the standard definition (character, summary, description, def
 
 # flagDescriptions.custom
 
-a flag with a custom typescript type. You'll need to add some code after the scaffolding is complete.
+Flag with a custom typescript type. You must add some code after the scaffolding is complete.
 
-# flagDescriptions.salesforceId
+# flagDescriptions.salesforceId 
 
-a valid salesforce record ID. You can specify the 3-character prefix and/or the length of the ID.
+A valid Salesforce record ID. You can specify the 3-character prefix and the length of the ID.
 
 # flagDescriptions.file
 
-a local file path. You can specify whether the file must exist.
+A local file. You can specify whether the file must already exist, or if the command will create it if necessary.
 
 # flagDescriptions.directory
 
-a local directory path. You can specify whether the file must exist.
+A local directory path. You can specify whether the directory must already exist, or if the command will create it if necessary.
 
 # flagDescriptions.orgApiVersion
 
-a valid salesforce API version number. The code checks for minimal non-retired values.
+A valid Salesforce API version number. The code checks for minimal non-retired values. You can optionally use the standard CLI API version flag ("--api-version").
 
 # flagDescriptions.requiredOrg
 
-a Salesforce org, entered by username/alias and aware of default org.
+A required Salesforce org, which a user specifies with either a Salesforce username or an alias. You can optionally use the standard CLI target org flag ("--target-org").
 
 # flagDescriptions.optionalOrg
 
-a Salesforce org, entered by username/alias and aware of default org.
+An optional Salesforce org, which a user specifies with either a Salesforce username or an alias. You can optionally use the standard CLI target org flag ("--target-org").
 
 # flagDescriptions.requiredHub
 
-a Salesforce org, entered by username/alias and aware of default dev hub. Must be a dev hub.
+A required Salesforce Dev Hub org, which a user specifies with either a Salesforce username or an alias. You can optionally use the standard CLI target Dev Hub flag ("--target-dev-hub").
 
 # flagDescriptions.optionalHub
 
-a Salesforce org, entered by username/alias and aware of default dev hub. Must be a dev hub.
+An optional Salesforce Dev Hub org, which a user specifies with either a Salesforce username or an alias. You can optionally use the standard CLI target Dev Hub flag ("--target-dev-hub").
 
 # flagDescriptions.url
 
-Validates that input matches URL spec. Returns the NodeJS Url class for simplified parsing.
+A valid URL. The code checks that the input matches the official W3C URL specification. Returns the NodeJS Url class for simplified parsing.
 
 # flagDescriptions.string
 
-Any valid string
+Any valid string.
 
 # flagDescriptions.boolean
 
-Take no value, has value of `true` if provided and `false` otherwise
+Value is either "true" (if the user specifies it when executing the command) or "false" (user doesn't specify it.)
```

Comment:
```suggestion
Value is either true (if the user specifies it when executing the command) or false (user doesn't specify it.)
```

If backticks aren't ok, let's take out the quotes so that it doesn't mean **string**.  They're true/false, not "true"/"false"

Reply:
```suggestion
Value is either `true` (if the user specifies it when executing the command) or `false` (user doesn't specify it.)
```